test/quantization/fx/test_quantize_fx.py

```diff
@@ -5000,7 +5000,7 @@ def forward(self, x):
 
         m = M()
         expected_node_occurrence = {
-            ns.call_module(torch.ao.quantization.FusedMovingAvgObsFakeQuantize): 6,
+            ns.call_module(torch.ao.quantization.FusedMovingAvgObsFakeQuantize): 5,
```

Comment:
is this expected?

Reply:
yeah, this PR also added support for all variants of add - torch.relu fusion, which exist partially before for (operator.add, torch.relu). This PR also supported (torch.add, torch.relu)